tutorials/raw/plot_object_raw.py

```diff
@@ -2,131 +2,557 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# The following table summarizes the various ways of subselecting data from a
+# :class:`~mne.io.Raw` object.
+#
+# .. cssclass:: table-bordered
+# .. rst-class:: midvalign
+#
+# +-------------------------------------+-------------------------+-----------+
+# | Python code                         | Result                  | Modifies  |
+# |                                     |                         | Raw       |
+# |                                     |                         | in-place? |
+# +=====================================+=========================+===========+
+# | ``raw.get_data()``                  | :class:`NumPy array     |           |
+# |                                     | <numpy.ndarray>`        | no        |
+# |                                     | (n_chans × n_samps)     |           |
+# +-------------------------------------+-------------------------+-----------+
+# | ``raw[:]``                          | :class:`tuple` of (data |           |
+# +-------------------------------------+ (n_chans × n_samps),    | no        |
+# | ``raw.get_data(return_times=True)`` | times (1 × n_samps))    |           |
+# +-------------------------------------+-------------------------+-----------+
+# | ``raw[0, 1000:2000]``               |                         |           |
+# +-------------------------------------+                         |           |
+# | ``raw['MEG 0113', 1000:2000]``      |                         |           |
+# +-------------------------------------+                         |           |
+# | ``raw.get_data(picks=0,             | :class:`tuple` of       |           |
+# | start=1000, stop=2000,              | (data (1 × 1000),       | no        |
+# | return_times=True)``                | times (1 × 1000))       |           |
+# +-------------------------------------+                         |           |
+# | ``raw.get_data(picks='MEG 0113',    |                         |           |
+# | start=1000, stop=2000,              |                         |           |
+# | return_times=True)``                |                         |           |
+# +-------------------------------------+-------------------------+-----------+
+# | ``raw[7:9, 1000:2000]``             |                         |           |
+# +-------------------------------------+                         |           |
+# | ``raw[[2, 5], 1000:2000]``          | :class:`tuple` of       |           |
+# +-------------------------------------+ (data (2 × 1000),       | no        |
+# | ``raw[['EEG 030', 'EOG 061'],       | times (1 × 1000))       |           |
+# | 1000:2000]``                        |                         |           |
+# +-------------------------------------+-------------------------+-----------+
+# | ``raw.pick_types(meg=False,         | :class:`~mne.io.Raw`    |           |
+# | eeg=True)``                         | object w/ only EEG      | yes       |
```

Comment:
This would also not be indexing in my book since indexing should not change the object.